tests/sentry/sentry_metrics/test_gen_metrics_multiprocess_steps.py

```diff
@@ -424,7 +424,7 @@ def test_process_messages_default_card_rollout(set_sentry_option: Callable[...,
     ),
     (
         {
-            "name": "c:transactions/alert@none" * 21,
+            "name": "c:transactions/alert@" + ("none" * 50),
```

Comment:
The test passed before because the MRI validation was not fully correct and it was accepting a wrong MRI. Now we pass a syntactically correct MRI but the unit is so long that it surpasses the 200 chars limit.